x-pack/plugins/ml/public/application/components/anomalies_table/links_menu.tsx

```diff
@@ -104,6 +109,7 @@ export const LinksMenuUI = (props: LinksMenuProps) => {
             },
           }
         : {}),
+      filters: getFiltersForDSLQuery(job.datafeed_config.query, null, job.job_id),
```

Comment:
❓ why don't we pass a data view id in this case? is it redundant for the maps view?

Reply:
Updated here https://github.com/elastic/kibana/commit/21b3a249907fc77d5dfca985a8d452911ef04d8b